src/components/SystemMessageNotification/SystemMessageNotification.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {useState} from 'react';
+import {Alert} from 'react-bootstrap';
+import styles from './SystemMessageNotification.module.scss';
+
+
+/*export function getSystemMessageType(type: Props) {
+  if (type.type === 'info') {
+    return 'info';
```

Comment:
I think what Jason was after here was to return the CSS class itself, not a string containing the name of the class. So on this line you would return `styles['info']`, on line 11 you would return `styles['warning']`, line 14 return `styles['notice']`, etc.

And then on line 32, instead of passing the style directly into your component, you'd call this function and pass it the type name. So `className={getSystemMessageType(props.type.toLowerCase())}`

This way if you ever needed to add new banner types or remove existing ones, you would just update the function to reflect the new types.

Bear in mind I'm not at home so I can't compile this, the bits of code I wrote here might not be typesafe or work exactly as written, but hopefully you get the idea. @jasekiw feel free to let me know if I'm barking up the wrong tree here.